Docs: Improve readability for README.md (#92)
Azure/co-op-translator: README.md

# Co-op Translator: Break Language Barriers Effortlessly

> _Easily automate translation into multiple languages for your project_

Reviewer: ```suggestion
_Easily automate the translation of your documentation into multiple languages_
```
